web/src/main.ts

```diff
@@ -1,9 +1,8 @@
-declare var define: any;
+import './lib/components/index.css';
```

Comment:
ruh roh, is this the beginning of putting css inside js?

Reply:
Well, that's just how Webpack works, we're completely innocent 😁